packages/hint-no-inline-styles/src/_locales/en/messages.json

```diff
@@ -0,0 +1,18 @@
+{
+    "elementsWithStyleAttributeFound": {
+        "description": "Message to show when an element with a style attribute is found",
+        "message": "CSS inline styles should not be used, move styles to an external CSS file"
+    },
+    "description": {
+        "description": "Metadata description",
+        "message": "Check if inline styles are used"
+    },
+    "styleElementFound": {
+        "description": "Message to show when a style element is found.",
+        "message": "CSS inline styles should not be used, move styles to an external CSS file"
```

Comment:
Both error messages are the same, is it intentional?

Reply:
No, it's not intentional. I missed it. Thanks for pointing it out, I will change it now.